### ISSUE TITLE
Anyone planning to update this for Cata?

### ISSUE THREAD

**Issue by scaredpork** [Aviana/LunaUnitFrames#980]
Just wondering if anyone was trying to get Luna running for the upcoming Cata classic pre-patch?

**Comment by gewoonnico** [comment]
Yeah was thinking the same thing, think a migration to SUF is the way to go :)

**Comment by liniim** [comment]
Unfortunately Luna seems to be broken for Cata and Era/HC. Looks like SUF it is. And its going to be much effort figuring out size and position settings from the Luna WTF settings.

**Comment by gewoonnico** [comment]
> Unfortunately Luna seems to be broken for Cata and Era/HC. Looks like SUF it is. And its going to be much effort figuring out size and position settings from the Luna WTF settings.

Someone posted a fix for SoD P5, which also works for the new Cata client: https://github.com/Aviana/LunaUnitFrames/issues/982#issuecomment-2372615062.

You can use this to temporarily fix Luna and swap over the positions with SUF. At least I know that's what I'm doing, Luna's going to keep breaking further with future game client updates.
